cmd/vc-rest/startcmd/start.go

```diff
@@ -30,31 +30,44 @@ import (
 )
 
 const (
-	hostURLFlagName          = "host-url"
-	hostURLFlagShorthand     = "u"
-	hostURLFlagUsage         = "URL to run the vc-rest instance on. Format: HostName:Port."
-	hostURLEnvKey            = "VC_REST_HOST_URL"
-	edvURLFlagName           = "edv-url"
-	edvURLFlagShorthand      = "e"
-	edvURLFlagUsage          = "URL EDV instance is running on. Format: HostName:Port."
-	edvURLEnvKey             = "EDV_REST_HOST_URL"
-	blocDomainFlagName       = "bloc-domain"
-	blocDomainFlagUsage      = "Bloc domain"
-	blocDomainEnvKey         = "BLOC_DOMAIN"
-	hostURLExternalFlagName  = "host-url-external"
-	hostURLExternalEnvKey    = "VC_REST_HOST_URL_EXTERNAL"
-	hostURLExternalFlagUsage = "Host External Name:Port This is the URL for the host server as seen externally." +
+	hostURLFlagName      = "host-url"
```

Comment:
A few of these flags were missing their shorthand equivalents. I added them.

I also noticed that there was no spacing between any of these, so I grouped them together based on their purpose.